Userland/Libraries/LibPDF/Page.cpp

```diff
@@ -13,6 +13,10 @@ namespace PDF {
 PDFErrorOr<ByteBuffer> Page::page_contents(Document& document) const
 {
     // Table 3.27 Entries in a page object on Contents:
+    // "If this entry is absent, the page is empty. [...]"
+    if (contents.is_null())
```

Comment:
Organizationally, I think it makes more sense for the commit to come before the `pdf` commit so it never has the chance to crash in the first place